Documentation/install/upgrade.rst

```diff
@@ -168,7 +168,9 @@ Kubernetes resources are updated accordingly to version you are upgrading to:
 
   .. group-tab:: Helm
 
-    Deploy Cilium release via Helm:
+    Deploy Cilium release via Helm (Running this command will overwrite the
+    existing cluster's `ConfigMap` which might not be ideal if you want to keep
+    your existing `ConfigMap`.)
```

Comment:
For micro releases, we should just use `--reuse-values`.

Reply:
oh, actually AFAIU since we don't specify any `--set`, `--reuse-values` is the default.